script.js

```diff
@@ -0,0 +1,21 @@
+const element = document.getElementById('hamNenu');
+element.addEventListener('click', () => {
+  document.getElementById('navMenu').style.width = '100%';
+});
+
+function closeNavMenu() {
+  document.getElementById('navMenu').style.width = '0';
+  document.getElementById('navxid').style.marginLeft = '0';
+}
+closeNavMenu();
```

Comment:
- [ ] Updating the dom through JavaScript is really expensive. Do you think maybe we can avoid calling this function here? I think the nav menu should already start closed when the page first loads.